SourceKitStressTester/Sources/StressTester/StressTester.swift

```diff
@@ -175,7 +175,23 @@ public class StressTester {
           return options.requests.contains(.testModule)
         }
       }
-      .divide(into: options.page.count)
+    
+    // There are certain situations where we would issue the same request twice
+    // e.g. once for the end of a token and then for the start of the next 
+    // token. That's a waste of time. Filter them out.
```

Comment:
Oh nice catch, do you know how much of a difference this makes?

Reply:
I don’t think it’s much. Unfortunately, I don’t remember the scenarios where this happens since it’s been a month or two since I made this fix locally. We’ll see if there is a significant improvement when the stress tester runs.